src/ResourceManager/Resources/Commands.Resources/Commands.Resources.csproj

```diff
@@ -232,6 +232,10 @@
       <Project>{c60342b1-47d3-4a0e-8081-9b97ce60b7af}</Project>
       <Name>Commands.Profile</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\ServiceManagement\RemoteApp\Commands.RemoteApp\Commands.RemoteApp.csproj">
+      <Project>{492d2af2-950b-4f2e-8079-8794305313fd}</Project>
```

Comment:
It used to pass all the tests with this in, so bringing it in. Removing fails many tests.